env.sample

```diff
@@ -27,10 +29,24 @@ export PH_OAUTH_CLIENT_ID=opsapp
 
 export PH_OAUTH_CLIENT_SECRET=Y2xpZW50Og=
 
+export PH_OAUTH_CLIENT_UUUID=Y2xpZW50Og=
+
 export PH_OAUTH_BASIC_AUTH=true
 
 export PH_OAUTH_BASIC_AUTH_TOKEN=Y2xpZW50Og==
 
+export PH_JBPM_API_URL=http://localhost:8180/kie-server/services/rest/server
```

Comment:
Which is the helm chart that must be deployed to this endpoint? Why it is pointing to localhost instead of a DNS of the API Gateway?

Reply:
So in the frontend we needed some tool for handling maker checker process of user creation so we have used a buisness process management called JBPM.